<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Homework: Lesson13</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabVector">
       <attribute name="title">
        <string>Вектор строк</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <widget class="QLabel" name="labelCountDescription">
          <property name="text">
           <string>Количество:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>549</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="lineEditPattern">
          <property name="placeholderText">
           <string>Паттерн</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="labelCount">
          <property name="text">
           <string>&lt;нет данных&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPushButton" name="pushButtonApplyPattern">
          <property name="text">
           <string>Применть</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayoutDescription">
          <item>
           <widget class="QLabel" name="labelInput">
            <property name="text">
             <string>Исходые данные</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelResult">
            <property name="text">
             <string>Результат</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayoutEditors">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEditInput">
            <property name="plainText">
             <string>год
человек
время
дело
жизнь
день
рука
раз
работа
слово
место
лицо
друг
глаз
вопрос
дом
сторона
страна
мир
случай
голова
ребенок
сила
конец
вид
система
часть
город
отношение
женщина
деньги
земля
машина
вода
отец
проблема
час
право
нога
решение
дверь
образ
история
власть
закон
война
бог
голос
тысяча
книга
возможность
результат
ночь
стол
имя
область
статья
число
компания
народ
жена
группа
развитие
процесс
суд
условие
средство
начало
свет
пора
путь
душа
уровень
форма
связь
минута
улица
вечер
качество
мысль
дорога
мать
действие
месяц
государство
язык
любовь
взгляд
мама
век
школа
цель
общество
деятельность
организация
президент
комната
порядок
момент
театр
письмо
утро
помощь
ситуация
роль
рубль
смысл
состояние
квартира
орган
внимание
тело
труд
сын
мера
смерть
рынок
программа
задача
предприятие
окно
разговор
правительство
семья
производство
информация
положение
центр
ответ
муж
автор
стена
интерес
федерация
правило
управление
мужчина
идея
партия
совет
счет
сердце
движение
вещь
материал
неделя
чувство
глава
наука
ряд
газета
причина
плечо
цена
план
речь
точка
основа
товарищ
культура
данные
мнение
документ
институт
ход
проект
встреча
директор
срок
палец
опыт
служба
судьба
девушка
очередь
лес
состав
член
количество
событие
объект
зал
создание
значение
период
шаг
брат
искусство
структура
номер
пример
исследование
гражданин
игра
начальник
рост
тема
принцип
метод
тип
фильм
край
гость
воздух
характер
борьба
использование
размер
образование
мальчик
кровь
район
небо
армия
класс
представитель
участие
девочка
политика
герой
картина
доллар
спина
территория
пол
поле
изменение
направление
рисунок
течение
церковь
банк
сцена
население
большинство
музыка
правда
свобода
память
команда
союз
врач
договор
дерево
факт
хозяин
природа
угол
телефон
позиция
двор
писатель
самолет
объем
род
солнце
вера
берег
спектакль
фирма
способ
завод
цвет
журнал
руководитель
специалист
оценка
регион
песня
процент
родитель
море
требование
основание
половина
роман
круг
анализ
стихи
автомобиль
экономика
литература
бумага
поэт
степень
господин
надежда
предмет
вариант
министр
граница
дух
модель
операция
пара
сон
название
ум
повод
старик
миллион
успех
счастье
ребята
кабинет
магазин
пространство
выход
удар
база
знание
текст
защита
руководство
площадь
сознание
возраст
участник
участок
пункт
линия
желание
папа
доктор
губа
дочь
среда
председатель
представление
солдат
художник
волос
оружие
соответствие
ветер
парень
зрение
генерал
огонь
понятие
строительство
ухо
грудь
нос
страх
услуга
содержание
радость
безопасность
продукт
комплекс
бизнес
сад
сотрудник
лето
курс
предложение
рот
технология
реформа
отсутствие
собака
камень
будущее
рассказ
контроль
река
продукция
сумма
техника
здание
сфера
необходимость
фонд
подготовка
лист
республика
хозяйство
воля
бюджет
снег
деревня
мужик
элемент
обстоятельство
немец
победа
источник
звезда
выбор
масса
итог
сестра
практика
проведение
карман
слава
кухня
определение
функция
войско
комиссия
применение
капитан
работник
обеспечение
офицер
фамилия
предел
выборы
ученый
бутылка
бой
теория
зона
отдел
зуб
разработка
личность
гора
товар
метр
праздник
влияние
читатель
удовольствие
актер
слеза
ответственность
учитель
акт
боль
множество
особенность
показатель
корабль
звук
впечатление
частность
детство
вывод
профессор
доля
норма
прошлое
командир
коридор
поддержка
рамка
враг
этап
черт
дед
собрание
прием
болезнь
клетка
кожа
заявление
попытка
сравнение
расчет
депутат
комитет
знак
дядя
учет
хлеб
чай
режим
целое
вирус
выражение
здоровье
зима
десяток
глубина
сеть
студент
секунда
скорость
поиск
суть
налог
ошибка
доход
режиссер
поверхность
ощущение
карта
клуб
станция
революция
колено
министерство
стекло
этаж
высота
бабушка
трубка
газ
мастер
поведение
столица
механизм
передача
способность
подход
энергия
существование
исполнение
кино
сожаление
заместитель
ресурс
акция
рождение
администрация
стоимость
улыбка
артист
сосед
фраза
фигура
субъект
реакция
список
фотография
журналист
май
нарушение
заседание
толпа
больница
существо
свойство
долг
поколение
животное
схема
усилие
отличие
остров
противник
волна
реализация
страница
формирование
житель
красота
птица
растение
тень
явление
храм
запах
водка
наличие
ужас
одежда
кресло
больной
поезд
университет
традиция
адрес
декабрь
ладонь
сведение
цветок
лидер
октябрь
занятие
сентябрь
помещение
январь
зритель
редакция
стиль
весна
фактор
август
известие
зависимость
охрана
оборудование
концерт
отделение
расход
выставка
милиция
переход
эпоха
запад
произведение
родина
собственность
тайна
трава
лагерь
имущество
кровать
аппарат
середина
март
клиент
дама
фронт
отрасль
стул
беседа
законодательство
продажа
повышение
музей
след
полковник
сомнение
понимание
апрель
князь
рыба
дума
кодекс
сутки
чудо
шея
судья
крыша
настроение
поток
должность
преступление
мозг
честь
пост
еврей
июнь
сотня
дождь
лестница
дача
установка
появление
получение
образец
труба
главное
осень
костюм
баба
ценность
обязанность
пьеса
таблица
вино
воспоминание
лошадь
коллега
организм
ученик
учреждение
открытие
том
черта
характеристика
выполнение
оборона
выступление
температура
перспектива
подруга
приказ
жертва
ресторан
километр
спор
вкус
признак
промышленность
американец
лоб
заключение
восток
исключение
ключ
постановление
слой
бок
июль
перевод
секретарь
кусок
слух
польза
звонок
обстановка
чиновник
соглашение
деталь
русский
тишина
зарплата
билет
подарок
тюрьма
ящик
конкурс
книжка
изучение
просьба
царь
публика
смех
сообщение
угроза
беда
блок
достижение
назначение
реклама
портрет
масло
стакан
урок
часы
крик
творчество
телевизор
инструмент
концепция
лейтенант
экран
дно
реальность
канал
мясо
знакомый
щека
конфликт
переговоры
запись
вагон
площадка
последствие
сотрудничество
зеркало
тон
академия
палата
потребность
ноябрь
увеличение
дурак
поездка
обед
потеря
февраль
мероприятие
парк
принятие
устройство
вещество
категория
сезон
гостиница
издание
объединение
темнота
человечество
колесо
опасность
разрешение
воздействие
коллектив
камера
запас
следствие
длина
крыло
округ
фон
кандидат
родственник
давление
присутствие
взаимодействие
доска
партнер
двигатель
шум
достоинство
грех
нож
полет
страсть
испытание
TRUE
799
разница
водитель
пакет
снижение
формула
живот
капитал
мост
новость
эффект
вход
губернатор
доклад
смена
убийство
эксперт
автобус
платье
кадр
тетя
общение
психология
лев
порог
проверка
процедура
рабочий
ремонт
обращение
обучение
ожидание
памятник
корень
наблюдение
буква
доказательство
признание
постель
штаб
владелец
компьютер
инженер
старуха
лодка
ракета
серия
шутка
вершина
выпуск
кулак
лед
торговля
нефть
молодежь
цифра
корпус
недостаток
сапог
сущность
талант
эффективность
кофе
полоса
основное
рассмотрение
сбор
штат
следователь
жилье
мешок
описание
куст
отказ
замок
редактор
дворец
забота
пиво
диван
столик
эксперимент
печать
кольцо
пистолет
воспитание
начальство
профессия
ворота
добро
дружба
покой
риск
окончание
дым
брак
величина
записка
инициатива
совесть
активность
кость
спорт
кредит
господь
майор
конференция
потолок
библиотека
помощник
конструкция
отдых
ручка
металл
молоко
прокурор
транспорт
поэзия
соединение
краска
расстояние
мечта
село
еда
зло
подразделение
сюжет
рубеж
сигнал
атмосфера
крест
вес
взрыв
контакт
сигарета
восторг
золото
почва
премия
король
подъезд
шанс
автомат
заказ
мальчишка
очки
миг
штука
чтение
поселок
свидетель
ставка
сумка
удивление
хвост
песок
поворот
возвращение
мгновение
статус
озеро
строй
параметр
сказка
тенденция
вина
дыхание
версия
масштаб
монастырь
хозяйка
дочка
танец
эксплуатация
коммунист
пенсия
приятель
объяснение
набор
производитель
пыль
философия
мощность
обязательство
уход
горло
кризис
указание
плата
яблоко
препарат
действительность
москвич
остаток
изображение
сделка
сочинение
покупатель
танк
затрата
строка
единица
обработка
чемпионат</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEditResult">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabPainter">
       <attribute name="title">
        <string>Случайный художник</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBoxDraw">
          <property name="title">
           <string>Холст</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdNumberLine"/>
           </item>
           <item row="1" column="1">
            <widget class="QLCDNumber" name="lcdNumberRectangle"/>
           </item>
           <item row="2" column="1">
            <widget class="QLCDNumber" name="lcdNumberCircle"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelLine">
             <property name="text">
              <string>Линии:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelRectangle">
             <property name="text">
              <string>Прямоугольники:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelCircle">
             <property name="text">
              <string>Круги:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="frameDrawActions">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxLine">
             <property name="text">
              <string>линия</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRect">
             <property name="text">
              <string>прямоугольник</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxCircle">
             <property name="text">
              <string>круг</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMath">
       <attribute name="title">
        <string>Математика</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="labelMathInput">
          <property name="text">
           <string>Введите данные для вычисления:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditMathInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonMath">
          <property name="text">
           <string>вычислить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMathResult">
          <property name="text">
           <string>Результат:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditMathResult">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
